charts/managed-identity-wallets/templates/_helpers.tpl

```diff
@@ -70,6 +70,13 @@ Usage: include "postgresContext" (list $ "your_include_function_here")
 {{- include $function (dict "Values" $.Values.postgresql "Chart" (dict "Name" "postgresql") "Release" $.Release) }}
 {{- end }}
 
+{{/*
+Create the default JDBC url
+*/}}
+{{- define "managed-identity-wallets.jdbcUrl" -}}
+{{- printf "jdbc:postgresql://%s-postgresql:5432/postgres?user=%s&password=%s" .Release.Name .Values.postgresql.secret.user .Values.postgresql.secret.user }}
```

Comment:
Shouldn't the last value be the password? It says `secret.user` twice

Reply:
yes it should be postgrespassword.